dsc/workflows/base/__init__.py

```diff
@@ -507,56 +510,6 @@ def get_bitstream_s3_uris(self, item_identifier: str) -> list[str]:
             item_identifier: The identifier used for locating the item's bitstreams.
         """
 
-    @final
```

Comment:
As functionality moves into `ItemSubmission`, I'm trying to be diligent about removing any redundant functionality from the `Workflow` class